rest_framework/fields.py

```diff
@@ -778,6 +778,8 @@ def to_internal_value(self, data):
         return data
 
     def to_representation(self, value):
+        if value is None:
+            return None
```

Comment:
Here `PrimaryKeyRelatedField` calls `self.pk_field.to_representation(value.pk)`.

When that's a `UUIDField` we hit this block:

```
        if self.uuid_format == 'hex_verbose':
            return str(value)
```

When `value` is `None`, without the fix, that's equivalent to:

```
>>> str(None)
'None'
```

which is the observed output.